### ISSUE TITLE
error with extract_clim in microclima-free version

### ISSUE THREAD

**Issue by admahood** [dklinges9/mcera5#27]
I think it's something to do with the internal nc_to_df function?

clim_point <- mcera5::extract_clim(nc = ncs,
                                                               d_weight = F,
                                                                long = x,                           
                                                                lat = y,
                                                                 start_time = mef_tr[1], 
                                                                   end_time = lubridate::as_datetime("2022-12-21 00:00:00"))

Error in `dplyr::mutate()`:
ℹ In argument: `temperature = dplyr::case_when(...)`.
Caused by error in `dplyr::case_when()`:
! Can't recycle `..1 (left)` (size 240) to match `..2 (right)` (size 220).
Run `rlang::last_trace()` to see where the error occurred.

**Comment by dklinges9** [comment]
Hi Adam, could you send your code so i can reproduce the example? And either the CDS query code or the corresponding ERA5 netcdf file. If easier can attach to email to dklinges9@gmail.com

**Comment by dklinges9** [comment]
Finally getting around to this (Adam and I chatted via email and the urgency was removed on his end-- and so it was on mine too!) and discovered that it was indeed a bug: coastal_correct assumed that the user's ERA5 netCDF represented full days (ie time series divisible by 24 hours), which shouldn't be an assumption. [New commit](https://github.com/dklinges9/mcera5/commit/a0b178b70a4b89ea2e0e918086e88b618c275262) now entails via more flexible to accommodate non-complete days.

Thanks Adam!
